creator-node/src/index.ts

```diff
@@ -146,4 +174,11 @@ const startApp = async () => {
     }
   })
 }
-startApp()
+
+if (cluster.isMaster) {
```

Comment:
looks like deprecated in favor of `isPrimary`

Reply:
yep but our node version doesn't support isPrimary - I tried

Reply:
ah